Pipfile

```
@@ -0,0 +1,21 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
```

I think this section should be updated.
```suggestion
url = "https://github.com/binary-com/python-deriv-api"
verify_ssl = true
name = "python-driv-api"
```